lib/daru/io/importers/excel.rb

```diff
@@ -4,19 +4,29 @@ module Daru
   module IO
     module Importers
       class Excel < Base
-        Daru::DataFrame.register_io_module :from_excel, self
+        Daru::DataFrame.register_io_module :from_excel do |*args|
+          if args.first.end_with? '.xlsx'
+            require 'daru/io/importers/excelx'
```

Comment:
Does this mean that a `require` will be called each time the `from_excel` method is called? That's inefficient. You should ideally perform all the requires when the gem is required or when a specific importer/exporter is required.